_site/about/index.html

```diff
@@ -76,7 +76,7 @@ <h2 class="section__header">The Team 2019-20</h2>
         </br>
 		<div class="cards">
           <div class="team card">
-            <div class="card__image" style="background-image: url('/img/team_19-20/rakshitha_pro.jpg')"></div>
+            <div class="card__image" style="background-image: url('/img/team_19-20/rakshitha_pro.jpeg')"></div>
```

Comment:
Rename actual file extension as `.jpg` instead.

Also remove `_pro` at the end for all the files.